### ISSUE TITLE
Get the Artwork

### ISSUE THREAD

**Issue by HeXaN-Prime** [sebbo2002/pyatv-mqtt-bridge#183]
Hi,

in pyatv there is the possibility to get the Artwork. Is it possible to integrate this in the mqtt bridge?

Thanks a lot.

**Comment by sebbo2002** [comment]
Of course, it would be theoretically possible, but there are some open questions, which is why this feature is currently not implemented:

- Currently, fetching artwork only works via `atvremote`, not via `atvscript`. This would not be a blocker, but the probability that the feature will break later is correspondingly higher
- `atvremote artwork_save` stores the cover as an image locally. Using MQTT you could think of several ways to transfer the file. You could transfer the file binary, as a base64 string or publish a URL. Of course, the problem with the latter would be that a server would have to be running. At least the first two paths should be implemented.
- As far as I know cover changes are not announced via `push_updates`. Here a logic would have to be implemented accordingly, which fetches the cover e.g. with title changes anew.

These points have currently caused me not to implement this feature. But of course I would be happy about a corresponding pull request.

**Comment by chrigu1981** [comment]
implementing the URL would be enough for me

**Comment by asherry99** [comment]
+1 for this for reconsideration in the future. I think bringing in Artwork would be be of huge value to the project. But I understand your trade off / considerations above. My use case is I use pyatv-mqtt to display and control whats playing within my HA system so bringing in Artwork is of high desire. 

My two cents, base64 of the image pushed over mqtt would be prefered over the URL as that feels simpler and more inline w/ a bridge than running a server. But I'd take any solution.

**Comment by SidM01** [comment]
+1 I would also like the function to transfer the image (preferably via a URL) and would be very happy

**Comment by asherry99** [comment]
@sebbo2002 Would be be possible to keep this Issue open (instead of the stale auto closure)? Maybe re-label it `Feature Request: Get the Artwork` or use a Feature Request GH labels to allow it stay open?

My hope is by it being an open issue and not closed one more people will see it and eventually it's enough of a desire/traction that someone runs with this? I think it would be a huge value add the the project, but understand there's still open questions and a decent amount of time that would go into implementing it. I just worry by it being closed it feels like a totally dead issue. 🤷🏻‍♂️
